src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/BaseStandardCredentials.java

```diff
@@ -45,12 +48,13 @@ protected BaseStandardCredentials(PageArea area, String relativePath) {
     }
 
     public void setId(String id) {
-        control("advanced-button").click();
-        control("id").set(id);
+        //control("advanced-button").click();
+        control(by.name("_.id")).set(id);
     }
     
     /**
      * Clicks the delete credential button and wait until it is not present anymore
+     * TO REMOVE - not possible with new credentials
```

Comment:
You're right, I missed deleting this with the second round of cleanups.  Removing...